include/topic_based_ros2_control/topic_based_system.hpp

```diff
@@ -89,6 +94,7 @@ class TopicBasedSystem : public hardware_interface::SystemInterface
   /// The size of this vector is (standard_interfaces_.size() x nr_joints)
   std::vector<std::vector<double>> joint_commands_;
   std::vector<std::vector<double>> joint_states_;
+  std::vector<std::vector<double>> initial_joint_commands_;
```

Comment:
Why do we need this?

Reply:
This is used to store the initial value of each joint; assigning to joint_commands_ will overwrite the value.

Reply:
When using Isaac Sim and Move It 2, I have a different variable for joint_command because of the overwriting behavior when I set the initial value to joint_command. It seems that joint_command is overwritten with all zeros even when joint_command messages are not exchanged.